frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-card/radiogram-card-content-personnel-count/radiogram-card-content-personnel-count.component.ts

```diff
@@ -0,0 +1,36 @@
+import type { OnInit } from '@angular/core';
+import { Component, Input } from '@angular/core';
+import { Store } from '@ngrx/store';
+import type { PersonnelCountRadiogram } from 'digital-fuesim-manv-shared';
+import { UUID } from 'digital-fuesim-manv-shared';
+import type { Observable } from 'rxjs';
+import type { AppState } from 'src/app/state/app.state';
+import { createSelectRadiogram } from 'src/app/state/application/selectors/exercise.selectors';
+
+@Component({
+    selector: 'app-radiogram-card-content-personnel-count',
+    templateUrl: './radiogram-card-content-personnel-count.component.html',
+    styleUrls: ['./radiogram-card-content-personnel-count.component.scss'],
+})
+export class RadiogramCardContentPersonnelCountComponent implements OnInit {
+    @Input() radiogramId!: UUID;
+
+    radiogram$!: Observable<PersonnelCountRadiogram>;
+
+    readonly personnelCategories = [
+        'notarzt',
+        'notSan',
+        'rettSan',
+        'san',
+        'gf',
+    ] as const;
```

Comment:
Could you please add this to `shared/src/models/utils/personnel-type.ts` and import it from there?
```ts
export const personnelTypes = StrictObject.keys(personnelTypeAllowedValues);
```

Its also used in the general tab, so it might also be sensible to change it there as well :)

Reply:
I did not want to compute this dynamically since the order of the elements in the array defines the order in which the elements are shown in the user interface. Placing the array in shared could cause the UI to change suddenly and it would be hard to track the cause.